core/src/main/java/com/sequenceiq/cloudbreak/converter/v2/FileSystemToFileSystemV2RequestConverter.java

```diff
@@ -0,0 +1,64 @@
+package com.sequenceiq.cloudbreak.converter.v2;
+
+import java.util.Map;
+
+import com.sequenceiq.cloudbreak.api.model.AdlsFileSystemConfiguration;
+import com.sequenceiq.cloudbreak.api.model.FileSystemConfiguration;
+import com.sequenceiq.cloudbreak.api.model.filesystem.AdlsFileSystemParameters;
+import com.sequenceiq.cloudbreak.api.model.filesystem.GcsFileSystemParameters;
+import com.sequenceiq.cloudbreak.api.model.filesystem.S3FileSystemParameters;
+import com.sequenceiq.cloudbreak.api.model.filesystem.WasbFileSystemParameters;
+import com.sequenceiq.cloudbreak.api.model.v2.FileSystemV2Request;
+import com.sequenceiq.cloudbreak.controller.exception.BadRequestException;
+import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
+import com.sequenceiq.cloudbreak.domain.FileSystem;
+
+public class FileSystemToFileSystemV2RequestConverter extends AbstractConversionServiceAwareConverter<FileSystem, FileSystemV2Request> {
+
+    @Override
+    public FileSystemV2Request convert(FileSystem source) {
+        FileSystemV2Request request = new FileSystemV2Request();
+        request.setDescription(source.getDescription());
+        if (source.getType() != null) {
+            Map<String, String> properties = source.getProperties();
+            switch (source.getType()) {
```

Comment:
Would be better to create separated converters for all, and call the converters from here